test/Spec.hs

```diff
@@ -1,2 +1,11 @@
+module Main where
```

Comment:
Not using the `hspec-discover` machinery because.. well because eventually I'll want to run the database tests in a more reasonable manner, and the `hspec-discover` processor can't be bothered to have options or customization.